helix-rest/src/main/java/org/apache/helix/rest/common/RestSystemPropertyKeys.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.helix.rest.common;
+
+public class RestSystemPropertyKeys {
+  // System property for request timeout
+  public static final String HTTP_TIMEOUT_MS = "http.timeout.ms";
```

Comment:
Is this changed? Like I said in previous thread, The property name is too generic. If this is set, it indicates that all http in helix should be using the config. However this is not the case. Should be narrowed down to helix rest or even stoppable.

Reply:
Changed it to REST_HTTP_TIMEOUT_MS.